test/e2e/specs/site-editor/multi-entity-saving.spec.js

```diff
@@ -41,7 +41,7 @@ test.describe( 'Site Editor - Multi-entity save flow', () => {
 		).toBeEnabled();
 		await expect(
 			page
-				.getByRole( 'region', { name: 'Save panel' } )
+				.getByRole( 'region', { name: 'Editor publish' } )
```

Comment:
Why is this change needed? The region is for saving not just publishing no?

Reply:
Yeah, but that's what we [call it ](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/editor-interface/index.js#L43)for a while.. Should we change it in a follow up?